include/envoy/upstream/upstream.h

```diff
@@ -322,6 +322,9 @@ class ClusterInfo {
   struct Features {
     // Whether the upstream supports HTTP2. This is used when creating connection pools.
     static const uint64_t HTTP2 = 0x1;
+    // Use the downstream protocol (HTTP1.1, HTTP2) for upstream connections as well, if available.
+    // This is used when creating connection pools.
+    static const uint64_t USE_DOWNSTREAM_PROTOCOL = 0x2;
```

Comment:
note: Needs name update per @wora name change in data-plane-api PR.

Reply:
Done.